django/contrib/gis/db/models/functions.py

```diff
@@ -367,15 +367,21 @@ class ForcePolygonCW(GeomOutputGeoFunc):
 
 
 class FromWKB(GeoFunc):
-    output_field = GeometryField(srid=0)
-    arity = 1
+    arity = 2
     geom_param_pos = ()
 
+    def __init__(self, expression, srid=0, **extra):
+        expressions = [
+            expression,
+            self._handle_param(srid, "srid", int),
+        ]
+        if "output_field" not in extra:
+            extra["output_field"] = GeometryField(srid=srid)
+        super().__init__(*expressions, **extra)
```

Comment:
`srid` is not supported on Oracle. We must ignore it, e.g.
```python
    def as_oracle(self, compiler, connection, **extra_context):
        source_expressions = self.get_source_expressions()
        clone = self.copy()
        clone.set_source_expressions(source_expressions[:1])
        extra_context["output_field"] = GeometryField(srid=0)
        return super(FromWKB, clone).as_sql(compiler, connection, **extra_context)
```

Reply:
If you want to test on Oracle, try [my django-docker-box fork main branch](https://github.com/ngnpope/django-docker-box/commits/main)… Will finish polishing it off eventually, but it already resolves many issues.